Content.Client/EntryPoint.cs

```diff
@@ -80,6 +80,7 @@ public override void Init()
             prototypes.RegisterIgnore("seed"); // Seeds prototypes are server-only.
             prototypes.RegisterIgnore("barSign");
             prototypes.RegisterIgnore("objective");
+            prototypes.RegisterIgnore("dataset");
```

Comment:
Do we really want to ignore this client-side? I imagine it'd be best to simply not ship any server-only datasets with the client

Reply:
this is due to the `DatasetPrototype` being in `Content.Server`. I could move it to shared if you want?

Reply:
This prototype should 100% be on the client as well.